src/models/Scholarships.ts

```diff
@@ -191,6 +192,11 @@ class Scholarships extends FirestoreCollection<ScholarshipData> {
                 data.requirements?.states,
                 opts.states
               ) &&
+              // school filter
+              requirementMatchesFilter(
+                data.requirements?.schools,
+                opts.schools
```

Comment:
Lower case these so we're not case sensitive. Or maybe `requirementMatchesFilter()` should do `.map((s) => s?.toLowerCase() || s)` on both lists so we lower case both sides when it's dealing with string lists.